core/modules/system/system.theme.inc

```diff
@@ -88,11 +88,31 @@ function theme_system_themes_page($variables) {
 
   $output = '<div id="system-themes-page">';
 
+  // @todo make these not dependant on admin_bar?
+  if (module_exists('admin_bar')) {
+    // If we have a cached menu for the current user, only output the hash
+    // for the client-side HTTP cache callback URL.
+    global $user, $language;
+    $cid = 'admin_bar:' . $user->uid . ':' . session_id() . ':' . $language->langcode;
+    if ($hash = admin_bar_cache_get($cid)) {
+      $options = array(
+        'attributes' => array('class' => array('button')),
+        'query' => array('token' => $hash, backdrop_get_destination()),
+      );
+      $output .= '<div class="system-themes-cache-buttons message-info">';
```

Comment:
Why isn't this output via `backdrop_set_message()`, with `info` as its type instead?

Reply:
Because it's not a message. It will always be on the page, regardless of previous actions taken.